### ISSUE TITLE
ValueError: insecure string pickle

### ISSUE THREAD

**Issue by lartkma** [simon-weber/gchatautorespond#11]
I downloaded the project on a Windows 10 machine, and I used Python 2.7.14 (There is no indication of which Python version this project uses, but I tried to execute it on Python 3 and I got a syntax error, so I suppose it's Python 2).

After executing the dependency script with `pip`, I tried to execute the `auth` command (as said in the README) and I got the following error:

```
Traceback (most recent call last):
  File "standalone_bot.py", line 27, in <module>
    django.setup()
  File "C:\Python27\lib\site-packages\django\__init__.py", line 17, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "C:\Python27\lib\site-packages\django\conf\__init__.py", line 48, in __getattr__
    self._setup(name)
  File "C:\Python27\lib\site-packages\django\conf\__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "C:\Python27\lib\site-packages\django\conf\__init__.py", line 92, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "C:\Python27\lib\importlib\__init__.py", line 37, in import_module
    __import__(name)
  File "C:\Users\my.user\git\gchatautorespond\gchatautorespond\settings_standalone.py", line 1, in <module>
    from .settings_dev import *  # noqa
  File "C:\Users\my.user\git\gchatautorespond\gchatautorespond\settings_dev.py", line 1, in <module>
    from .settings import *  # noqa
  File "C:\Users\my.user\git\gchatautorespond\gchatautorespond\settings.py", line 35, in <module>
    TEST_CREDENTIALS = pickle.loads(base64.b64decode(get_secret('test_account.credentialrow')))
  File "C:\Python27\lib\pickle.py", line 1388, in loads
    return Unpickler(file).load()
  File "C:\Python27\lib\pickle.py", line 864, in load
    dispatch[key](self)
  File "C:\Python27\lib\pickle.py", line 976, in load_string
    raise ValueError, "insecure string pickle"
ValueError: insecure string pickle
```

**Comment by szilviu** [comment]
```
Traceback (most recent call last):
  File "standalone_bot.py", line 27, in <module>
    django.setup()
  File "/usr/local/lib/python2.7/dist-packages/django/__init__.py", line 17, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 48, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 92, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/var/www/html/gchatautorespond/gchatautorespond/settings_standalone.py", line 1, in <module>
    from .settings_dev import *  # noqa
  File "/var/www/html/gchatautorespond/gchatautorespond/settings_dev.py", line 1, in <module>
    from .settings import *  # noqa
  File "/var/www/html/gchatautorespond/gchatautorespond/settings.py", line 35, in <module>
    TEST_CREDENTIALS = pickle.loads(base64.b64decode(get_secret('test_account.credentialrow')))
  File "/usr/lib/python2.7/pickle.py", line 1388, in loads
    return Unpickler(file).load()
  File "/usr/lib/python2.7/pickle.py", line 864, in load
    dispatch[key](self)
  File "/usr/lib/python2.7/pickle.py", line 976, in load_string
    raise ValueError, "insecure string pickle"
ValueError: insecure string pickle
```
Same. Python 2.7.12, Ubuntu 16.04.4

**Comment by boxofdeath** [comment]
Same here. Python 2.7.15rc1

**Comment by jsswope** [comment]
I also get:

File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/pickle.py", line 966, in load_string
    raise ValueError, "insecure string pickle"
ValueError: insecure string pickle

when trying to import:
import matplotlib.pyplot as plt

Python interpreter:
Python 2.7.10 (default, Oct  6 2017, 22:29:07) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)] on darwin

**Comment by simon-weber** [comment]
Sorry, I messed up my github settings and never saw this issue! It should be fixed in 47de57769b4dd8c5075160aad2e11ef749599778.
